From FEDORA_PATCHES Mon Sep 17 00:00:00 2001
From: Elena Zannoni <ezannoni@redhat.com>
Date: Fri, 27 Oct 2017 21:07:50 +0200
Subject: gdb-6.3-rh-dummykfail-20041202.patch

FileName: gdb-6.3-rh-dummykfail-20041202.patch

;; Work around out-of-date dejagnu that does not have KFAIL
;;=push: That dejagnu is too old to be supported.

2003-07-11  Elena Zannoni  <ezannoni@redhat.com>

        * lib/gdb.exp (setup_kfail, kfail): Redefine procedures.
---
 gdb/testsuite/lib/gdb.exp | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/gdb/testsuite/lib/gdb.exp b/gdb/testsuite/lib/gdb.exp
index 7702d9c238..9884f004c5 100644
--- a/gdb/testsuite/lib/gdb.exp
+++ b/gdb/testsuite/lib/gdb.exp
@@ -124,6 +124,15 @@ set valnum_re "\\\$$decimal"
 
 ### Only procedures should come after this point.
 
+if {![llength [info procs kfail]]} {
+    proc setup_kfail { args } {
+	#setup_xfail args
+    }
+    proc kfail { bugid message } {
+	fail $message
+    }
+}
+
 #
 # gdb_version -- extract and print the version number of GDB
 #
-- 
2.14.3

